contracts/tgrade-validator-voting/src/contract.rs

```diff
@@ -284,7 +284,11 @@ fn privilege_change(change: PrivilegeChangeMsg) -> Response {
 
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn migrate(deps: DepsMut, _env: Env, _msg: Empty) -> Result<Response, ContractError> {
-    ensure_from_older_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
+    ensure_from_older_version(
+        deps.storage,
+        "crates.io:tgrade_validator_voting_proposals",
```

Comment:
That's the current name?

Reply:
It is, up to and including v0.11.

This will require a reversion after this version is deployed / migrated to. We can always do that after this release.